zipline/__main__.py

```diff
@@ -17,6 +18,20 @@
 except NameError:
     __IPYTHON__ = False
 
+extension_args = {}
```

Comment:
I think we should make this dictionary available at either the top level like `zipline.extension_args` or somewhere easy like `zipline.extensions.extension_args`.

@ssanderson do you have any ideas?

Reply:
I don't have a super strong preference about whether this is a a dictionary or a namespace object. Do we think that most extensions are going to know ahead of time what args they're interested in? If so then a namespace object probably makes sense. If we think people are going to need to dynamically scan the args, then a dict is probably preferable.